src/gui/properties/trackerlistwidget.cpp

```diff
@@ -598,10 +598,10 @@ void TrackerListWidget::showTrackerListMenu(const QPoint &)
 
     if (!torrent->isPaused())
     {
-        menu->addAction(UIThemeManager::instance()->getIcon("view-refresh"), tr("Force reannounce to selected trackers")
+        menu->addAction(UIThemeManager::instance()->getIcon("reannounce"), tr("Force reannounce to selected trackers")
```

Comment:
You shouldn't change icon names since it breaks existing custom themes.

Reply:
I had to as "view-refresh" icon is used for both Force recheck and Force reannounce Icon. They server different purposes and thus, another icon was created with a new name.
@jagannatharjun Can you kindly make a patch in your theme bundle if changes were accepted?

Reply:
From this:
![image](https://user-images.githubusercontent.com/22009014/132989657-68bfa40f-e36d-4f45-b92a-bc2634e48048.png)
To this:
![image](https://user-images.githubusercontent.com/22009014/132989698-70ef2495-a384-4d6f-890c-6e7f3202e8bb.png)

Reply:
On the other hand, if it enriches the UI appearance, then we can allow such breaking changes. 
In addition, I would return to the issue of making custom themes "unpacked" in order to solve the compatibility problem between different Qt versions. @jagannatharjun, are you ready to do this now in order to make all the breaking changes related to the custom themes in the same release?

Reply:
> are you ready to do this now in order to make all the breaking changes related to the custom themes in the same release

Have exams, Will see if I get time, next month for sure 👍